src/handlers/group/group_handler.py

```diff
@@ -4,19 +4,25 @@
 from telegram.ext import ContextTypes
 from typing import Optional
 
-from sqlalchemy import select
+from sqlalchemy import select, func
 
 from src.logging import tg_logger
 from src import constants
 from src.texts import _
 from src.model import Chat, User, session_scope
-from src.handlers.utils import setup_counter
+from src.handlers.utils import setup_counter, setup_histogram
 
 
 new_member_counter = setup_counter("new_member.meter", "new_member_counter")
 whois_counter = setup_counter("new_whois.meter", "new_whois_counter")
 ban_counter = setup_counter("ban.meter", "ban_counter")
 
+async def db_metrics_reader_helper(context: ContextTypes.DEFAULT_TYPE):
+    chats_histogram = setup_histogram("chats.meter", "chats_counter")
+    async with session_scope() as sess:
+        result = await sess.execute(select(func.count(Chat.id)))
```

Comment:
we could also have another metric for `User` table size, maybe?
Both total User and unique user_id